### PR TITLE
Update fibonacci_sequence_recursion.py

### PR DESCRIPTION
**Pull request by NikhilCodes**
Fixed Missing `return` statement.

### CODE REVIEW

File: maths/fibonacci_sequence_recursion.py [hunk 1]
```diff
@@ -4,7 +4,7 @@ def recur_fibo(n):
     if n <= 1:
         return n
     else:
-        (recur_fibo(n-1) + recur_fibo(n-2))
+        return recur_fibo(n-1) + recur_fibo(n-2)
```
Comment (cclauss) [inline maths/fibonacci_sequence_recursion.py]: `return n if n <= 1 else recur_fibo(n-1) + recur_fibo(n-2)`

File: maths/fibonacci_sequence_recursion.py [hunk 2]
```diff
@@ -1,10 +1,7 @@
 # Fibonacci Sequence Using Recursion
 
 def recur_fibo(n):
-    if n <= 1:
```
Comment (cclauss) [inline maths/fibonacci_sequence_recursion.py]: Please add this __doctest__:
```
def recur_fibo(n):
    """
    >>> [recur_fibo(i) for i in range(12)]
    [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
    """
    return n if n <= 1 else recur_fibo(n-1) + recur_fibo(n-2)

File: maths/fibonacci_sequence_recursion.py [hunk 3]
```diff
@@ -1,10 +1,7 @@
...
-        return n
-    else:
-        (recur_fibo(n-1) + recur_fibo(n-2))
+    return n if n <= 1 else recur_fibo(n-1) + recur_fibo(n-2)
 
 def isPositiveInteger(limit):
```
Comment (cclauss) [inline maths/fibonacci_sequence_recursion.py]: This function is too simple to be useful.  Please removed it and do the test inline.

**Comment by NikhilCodes** [comment]
I just lost the forked-version hence cannot change (New GitHub user mistakes)
Hence I request reviewer to not mark it spam (Reason: Hacktober).
But I have created another branch to make changes intended to this branch.
New Branch: https://github.com/TheAlgorithms/Python/pull/1287

**Pull request closed without merging**